### PR TITLE
Translate content/id/docs/concepts/cluster-administration/cluster-administration-overview.md into Portuguese

### PR DESCRIPTION
**Pull request by devlware**
Hello @jcjesus @femrtnz!
First PR for the ptBR translation of the Kubernetes website. Can you please take a look?

Regards, Diego

### CODE REVIEW

File: content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+---
+
+{{% capture overview %}}
+A visão geral da administração de cluster é para qualquer um criando ou administrando um cluster Kubernetes.  Assume alguma familiaridade com os [conceitos](/docs/concepts/) centrais do Kubernetes.
+{{% /capture %}}
+
+{{% capture body %}}
+## Planejando um cluster
+
+Veja os guias em [Setup] (/docs/setup/) para exemplos de como planejar, configurar e configurar
```
Comment (femrtnz) [inline content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md]: I believe you need remove the space here `[Setup] (/docs/setup/)`

File: content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+## Planejando um cluster
+
+Veja os guias em [Setup] (/docs/setup/) para exemplos de como planejar, configurar e configurar  
+clusters Kubernetes. As soluções listadas neste artigo são chamadas *distros*.
+
+Antes de escolher um guia, aqui estão algumas considerações.
+
+
+- Você quer experimentar o Kubernetes no seu computador, ou você quer construir um cluster de alta  disponibilidade e multi-nós? Escolha as distros mais adequadas às suas necessidades.
+ - **Se você esta projetando para alta-disponibilidade**, aprender sobre configuração [clusters em múltiplas zonas](/docs/concepts/cluster-administration/federation/).
```
Comment (femrtnz) [inline content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md]: Maybe `aprenda` instead of `aprender` ?

Comment (jesus-amido) [reply content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md]: My suggestion is should be "saiba mais"

File: content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md (hunk 1 above)
Comment (jesus-amido) [reply content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md]: Diego, please, remove the duplicated word "configurar".

Good job!

Comment (devlware) [reply content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md]: @femrtnz @jesus-amido Thanks for reviewing.
For this sentence I feel that I missing something. I put the two 'configurar' on purpose so we can discuss the meaning of 'set up'.  The original says:  
"See the guides in [Setup](/docs/setup/) for examples of how to plan, __set up__, and configure Kubernetes clusters."

Do you agree if I say: "...para exemplos de como planejar, __ativar__ e configurar ..." 

Can you come up with another verb for __set up__?

Comment (jesus-amido) [reply content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md]: Looking at Google Translator result, I think we can use "**iniciar**" for set up.

See this link with search at GT.
[translate result](https://translate.google.com.br/#view=saved&op=translate&sl=en&tl=pt&text=set%20up)

Comment (femrtnz) [reply content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md]: I would go with "Olha o guia de Setup para exemplos de como planejar, configurar ...."

**Comment by devlware** [comment]
Sorry for the late delay guys! I'll finish all the corrections today ASAP.

File: content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+---
+reviewers:
+- jcjesus
+- femrtnz
```
Comment (femrtnz) [inline content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md]: I believe those are the content reviewers, not the translation reviewers. Maybe we should keep both here.

Comment (devlware) [reply content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md]: I fixed. Tks Felipe.

**Pull request merged**